### ISSUE TITLE
Feature: Combined Searches

### ISSUE THREAD

**Issue by mhammerschmidt** [Nayjest/Grids#163]
We want to implement combined search boxes which search across multiple columns.

For example

Columns Name, City

Enter "Homer" in a combined field "Search"

Searches in Both columns (WHERE name like ? OR city like ?).

We believe that is currently not possible.

Do you plan such a feature? How can it be implemented? Can you give us some hints? We would be happy to make a pull request then.

**Comment by wdog** [comment]
you can  use something like

```php
(new FilterConfig)
->setFilteringFunc(function ($val, EloquentDataProvider $provider) {
   $provider->getBuilder()->where('first_field, '=', $val).....
}),

**Comment by mhammerschmidt** [comment]
Yes, thanks we also came up with a similar idea... we then might add something like a colspan so the filter is spanning across columns. Or maybe the same filter should be added to all those affected rows with some kind of grouping id?
